FlowCal/plot.py

```diff
@@ -917,8 +1000,8 @@ def density2d(data,
             cbar.ax.set_ylabel('Counts')
 
     # Set scale of axes
-    plt.gca().set_xscale(xscale)
-    plt.gca().set_yscale(yscale)
+    plt.gca().set_xscale(xscale, data=data_plot, channel=0)
+    plt.gca().set_yscale(yscale, data=data_plot, channel=1)
```

Comment:
ahh perfect. good to see that other existing scales already use kwargs. excellent extension of that